sympy/ntheory/tests/test_generate.py

```diff
@@ -195,10 +206,12 @@ def test_randprime():
 
 def test_primorial():
     assert primorial(1) == 2
-    assert primorial(1, nth=0) == 1
+    assert primorial(1, 0) == 1
```

Comment:
Why did you remove the keyword argument from the tests. IMO nth ought to be a keyword-only argument (and the value here should be a boolean, not an integer).

Reply:
I tried defining the eval of  primorial as ```def eval(cls, n, *, nth=True):``` to keep nth as keyword only argument. But it is not working. ValueError gets raised.
```
>>> primorial(1, nth=False)
  File "/home/faisal/sympy/sympy/core/function.py", line 278, in __new__
    raise ValueError("Unknown options: %s" % options)
ValueError: Unknown options: {'nth': True}
```

Yes, nth should a boolean.

Reply:
Now it is working on python shell but raises error when run on test file.